espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/UiautomatorParams.java

```diff
@@ -0,0 +1,26 @@
+package io.appium.espressoserver.lib.model;
+
+import javax.annotation.Nullable;
+
+public class UiautomatorParams extends AppiumParams {
+    private String byMethod;
+    private String value;
```

Comment:
can we have a better name for this variable?

Reply:
how about locator?

Reply:
makes sense